pkgs/by-name/li/libiff/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
```

Comment:
Could you please follow the RFC coding style?

`nix-shell -I nixpkgs=$PWD -p nixfmt-rfc-style --run 'nixfmt pkgs/by-name/li/libiff/package.nix'`